cmd2/argparse_completer.py

```diff
@@ -535,7 +565,10 @@ def _print_action_help(self, action: argparse.Action) -> None:
 
             prefix = '{}{}'.format(flags, param)
         else:
-            prefix = '{}'.format(str(action.dest).upper())
+            if action.dest != SUPPRESS:
```

Comment:
Does this PR close Issue #407?  If so, you should mark it as doing so in the description so that it auto-closes on merge.